packages/core/useUrlSearchParams/index.ts

```diff
@@ -0,0 +1,47 @@
+import { computed, Ref, ref, watch } from 'vue-demi'
+import { useBrowserLocation } from '..'
+import { useEventListener } from '../useEventListener'
+
+export function useUrlSearchParams(method: 'history'|'hash'): Ref<URLSearchParams> {
+  const location = useBrowserLocation()
```

Comment:
I think we don't need to use this hook, just read them from `window.location` should be good.